### ISSUE TITLE
Unable to load addon script from path: 'res://addons/py4godot/py4godot.gd'.

### ISSUE THREAD

**Issue by Manik2607** [niklas2902/py4godot#78]
got this error while enabling the extension.

**Comment by Manik2607** [comment]
its turned off but its working.
is this intended?

**Comment by niklas2902** [comment]
Hey,
thanks for reporting this. This is actually a failed experiment. I will remove this in later versions. Is this a problem? Would it be useful to turn it on and off?

**Comment by Manik2607** [comment]
> Hey,
> thanks for reporting this. This is actually a failed experiment. I will remove this in later versions. Is this a problem? Would it be useful to turn it on and off like that?
Generally you turn on extensions to use them I was also trying to do that.
But if it's possible to remove thate check box you should do it.
Or mention that turning on extension is not important.

**Comment by JakeTurnick** [comment]
Just finding this discussion - Does this mean the extension doesn't have to be enabled to work? I had the same issue using the Godot AssetLib to install but couldn't activate.

Side question - does this addon make Godot recognize .py files across the whol project?

**Comment by niklas2902** [comment]
Hey, 
yes, the plugin should work immediatly and recognize .py files accross the whole project. It might be, that after the first time downloading and installing, you have to close and reopen the editor. But afterwards it should work. 
Do you have trouble with getting it to work?

**Comment by JakeTurnick** [comment]
Yes the asset lib install works! Maybe a note saying you don't need to enable it in the plugins menu is worth adding to the top of the description - avoids confusion with Godot convention on plugins.

**Comment by niklas2902** [comment]
Yes, that's a good point. Thank you for your feedback
